dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -487,12 +488,13 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
     @ApiOperation(value = "Refresh all workflows owned by the authenticated user with specified organization.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Workflow.class, responseContainer = "List")
     public List<Workflow> refreshWorkflowsByOrganization(@ApiParam(hidden = true) @Auth User authUser,
             @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId,
-            @ApiParam(value = "Organization", required = true) @PathParam("organization") String organization) {
+            @ApiParam(value = "Organization", required = true) @PathParam("organization") String organization,
+            @ApiParam(value = "hard refresh", defaultValue = "true") @QueryParam("hardRefresh") @DefaultValue("true") Boolean hardRefresh) {
```

Comment:
I thought to default to hard refresh would be good because it maintains the current way of doing things. I was thinking that skipping specific versions would be the odd case because of this. 

It may be that it is more likely this will be called on its own, instead of being called after an update to the workflow or workflow version, in which case it might make more sense to default just do a soft refresh.